reverse_engineering/helpers/connections/keyPairConnection.js

```diff
@@ -1,6 +1,20 @@
+const fs = require('fs');
+const crypto = require('crypto');
 const { connectWithTimeout } = require('./connection.js');
+const errorCodes = require('../../../common/errorCodes.js');
 
 const authByKeyPair = ({ account, role, timeout, username, privateKeyPath, privateKeyPass }) => {
+	if (
+		!isValidPrivateKey({
+			privateKeyPath,
+			privateKeyPass,
+		})
+	) {
+		return Promise.reject({
```

Comment:
Please, create custom error object, that externds the `Error` and has some message that you want